.gitignore

```diff
@@ -88,4 +88,7 @@ typings/
 .fusebox/
 
 # DynamoDB Local files
-.dynamodb/
\ No newline at end of file
+.dynamodb/
+
+# IDE VSCode settings files
+.vscode/
```

Comment:
Specific ides should go in your global gitignore, rather than in every project you touch.